daprdocs/content/en/js-sdk-docs/js-client/_index.md

```diff
@@ -294,14 +304,114 @@ async function start() {
   const topic = "topic-a";
 
   // Configure Subscriber for a Topic
-  await server.pubsub.subscribe(pubSubName, topic, async (data: any) => console.log(`Got Data: ${JSON.stringify(data)}`));
+  // Method 1: Direct subscribtion through the `subscribe` method
+  await server.pubsub.subscribe(pubSubName, topic, async (data: any) => console.log(`Received Data: ${JSON.stringify(data)}`));
 
+  // Method 2: Direct susbcription with options through the `subscribeWithOptions` method
+  await server.pubsub.subscribeWithOptions(pubSubName, topic, { 
+    callback: async (data: any) => console.log(`Received Data: ${JSON.stringify(data)}`) 
+  });
+
+  // Method 3: Subscription afterwards through the `susbcribeOnEvent` method
+  await server.pubsub.subscribeWithOptions('pubsub-redis', 'topic-options-1', {});
```

Comment:
nit: Shall we add pubSubName/Topic to variable or use the existing one?

Reply:
I would keep it as this